### DISCUSSION TITLE
Capacity Definition Discrepancy In Ecker2015 Parameter Set

### DISCUSSION THREAD

**Discussion by MirAbbasAli2A** [pybamm-team/PyBaMM#3727] [Q&A]
Hello PyBamm Team - 

Could someone please explain to me why the nominal cell capacity in the [Ecker2015 parameter set](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/parameters/lithium_ion/Ecker2015.py) is 0.15625 [A.h]. The Kokam SLPB 75106100 cell that it models has been shown in [Ecker's paper](https://iopscience.iop.org/article/10.1149/2.0551509jes) and many other literature sources to have a capacity of 7.5 A.h. This discrepancy I believe is causing me to have the Q_p does not equal Q_Li warning where the total lithium capacity is not matching the cathode capacity. Can anyone please give me an explanation for this discrepancy? It would be greatly appreciated.

**Comment by agriyakhetarpal** [comment]
Hi @MirAbbasAli2A, [this issue](https://github.com/pybamm-team/PyBaMM/issues/2152#issuecomment-1477680998) might be similar. We would welcome a PR if it's possible for you to fix this, though I will leave it to others to comment on the specifics of the parameter set in depth.
